ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/internal/methods/permissioning/PermRemoveNodesFromWhitelist.java

```diff
@@ -68,6 +68,9 @@ public JsonRpcResponse response(final JsonRpcRequest req) {
               return new JsonRpcErrorResponse(req.getId(), JsonRpcError.WHITELIST_PERSIST_FAILURE);
             case ERROR_WHITELIST_FILE_SYNC:
               return new JsonRpcErrorResponse(req.getId(), JsonRpcError.WHITELIST_FILE_SYNC);
+            case ERROR_CANT_REMOVE_BOOTNODE:
```

Comment:
I don't love having the contraction in the constant name - do we have a convention on this? maybe change to cannot?

Reply:
or maybe use "prevent"

Reply:
ERROR_PREVENT_BOOTNODE_REMOVAL ?

Reply:
I think I'd rather use ERROR_CANNOT_REMOVE_BOOTNODE. Is that ok?

Reply:
`ERROR_BOOTNODE_CANNOT_BE_REMOVED`